Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean

```diff
@@ -1471,7 +1471,7 @@ This is the affine version of `Submodule.inclusion`. -/
 def inclusion (h : S₁ ≤ S₂) : S₁ →ᵃ[k] S₂ where
   toFun := Set.inclusion h
   linear := Submodule.inclusion <| AffineSubspace.direction_le h
-  map_vadd' _ _ := rfl
+  map_vadd' := fun ⟨_,_⟩ ⟨_,_⟩ => rfl
```

Comment:
These were already problematic on `master`. The kernel was doing a lot of work without telling it explicitly to expand the subtypes. It got worse with the changes to instance shapes. Since it is the kernel, there is not a ton of visibility as to the steps for further probing